### PR TITLE
tasks refactoring

### PR DESCRIPTION
**Pull request by petrjasek**
- [x] group tasks by stage and not by status
- [x] drop stage/status mapping in desks config
- [x] add published board showing published content on desk
- [x] add planned board showing scheduled content

### CODE REVIEW

File: client/app/scripts/superdesk-desks/views/desk-config-modal.html [hunk 1]
```diff
@@ -54,12 +54,12 @@ <h3 ng-show="desk.edit._id" translate>Edit "{{desk.edit.name}}" Desk</h3>
...
 						<div class="content">
 							<div class="stages-list clearfix" ng-class="{'edit-mode': editStage}">
-								<div ng-repeat="stage in stages">
+								<div ng-repeat="stage in stages track by stage._id">
 									<span ng-if="isActive(stage)" sd-character-count data-item="editStage.name" data-limit="limits.stage"></span>
 									<div class="stage" ng-click="select(stage)"
 										ng-class="{active: selected === stage, editable: isActive(stage)}">
 										<span ng-if="!isActive(stage)">{{ stage.name }}
-											<label class="label globalread" ng-if="!(stage.is_visible || stage.is_visible == null)">GLOBAL READ OFF</label>
+											<label class="label globalread" ng-if="!(stage.is_visible || stage.is_visible == null)" translate>GLOBAL READ OFF</label>
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/views/desk-config-modal.html]: Can we reduce indentation in this file from 8 to 4 (or maybe even 2) characters per level? Currently a lot of horizontal scrolling is required, making it more difficult to both review the code and edit it in the editor on less wide screens.

Comment (petrjasek) [inline client/app/scripts/superdesk-desks/views/desk-config-modal.html]: sure, 8 is way too much

**Comment by amagdas** [comment]
:+1:

**Comment by eozan** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**